resources/service-catalog-addons/charts/service-binding-usage-controller/templates/cluster-role.yaml

```diff
@@ -17,7 +17,7 @@ rules:
 - apiGroups: ["serving.knative.dev"]
   resources: ["services"]
   verbs: ["get", "list", "watch", "patch", "update"]
-- apiGroups: ["kubeless.io"]
+- apiGroups: ["serverless.kyma-project.io"]
```

Comment:
As the migration hook is _post_ install, will it cause any problems with controller?

Reply:
It's copied from Rafter migration. I will remove it.